aiida/backends/tests/engine/test_utils.py

```diff
@@ -30,8 +31,10 @@ def setUpClass(cls, *args, **kwargs):
         cls.authinfo = orm.AuthInfo(computer=cls.computer, user=orm.User.objects.get_default())
         cls.authinfo.store()
 
-    def test_exponential_backoff_success(self):
+    @staticmethod
+    def test_exp_backoff_success():
         """Test that exponential backoff will successfully catch exceptions as long as max_attempts is not exceeded."""
+        global ITERATION
```

Comment:
this is a bit confusing - can't you just remove the module-level `ITERATION` and get rid of the `global` here?